suiteData.js

```diff
@@ -12,8 +15,14 @@ class SuiteData {
             throw new Error("A suite 'name' parameter should be included on the query string or in the request body.");
         }
         var tests = obj && obj.tests;
+        if (_.isString(tests) && fs.existsSync(sanitize(tests))) {
```

Comment:
although you are using sanitize-filename, I still think a prefix should be added to the path the user sends. meaning, we should look for the suite the user is asking for under a folder named "suites"